airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryUtils.java

```diff
@@ -182,14 +182,24 @@ private static void attemptCreateTableAndTestInsert(final BigQuery bigquery, fin
     }
   }
 
-  public static Table createTable(final BigQuery bigquery, String datasetName, String tableName, Schema schema) {
+  public static Table createTable(final BigQuery bigquery, final String datasetName, final String tableName, final Schema schema) {
```

Comment:
It looks like the only place where we use this method is in `attemptCreateTableAndTestInsert`, which is called during `CHECK` operation. Why wouldn't we use `createPartitionedTable` instead of this method?